web/portal/templates/doc/how_to_acknowledge.md

```diff
@@ -21,3 +21,11 @@ If you use the `GCRCatalogs` package in a publication,
 we ask that you cite the following publication.
 
 * [DESCQA Paper](https://ui.adsabs.harvard.edu/abs/2018ApJS..234...36M/abstract)
+
+### Mirrored Data
```

Comment:
How about "Mirroring data for community/non-personal use"?  Just want to make the title a bit more descriptive.

Reply:
Yes, this should be more specific.